main.js

```diff
@@ -36,6 +36,48 @@ var formatReplacementsMap = {
 	a: '(date.getHours() >= 12 ? "pm" : "am")' // pm
 };
 
+function ODate(rootEl) {
```

Comment:
`Date` already exists in the JS language, is that possible to use that name still?